checkpointctl.go

```diff
@@ -84,6 +85,12 @@ func setupShow() *cobra.Command {
 		false,
 		"Display all additional information about the checkpoints",
 	)
+	flags.StringVar(
+		&format,
+		"format",
+		"table",
+		"Format the output using one of the following views: table, tree, or json (default: \"tree\").",
```

Comment:
I had suggested changing the default to "tree" since we were previously using that. If we're going to use "table" as the default, we should change it here too.

Reply:
Thanks @snprajwal, I forgot to update the description.